src/scss/constants.scss

```diff
@@ -112,6 +112,7 @@ $font-size--xxl: 30px;
 $font-size--xxxl: 50px;
 
 // Placeholders, until we can find meaningful names for them
+$font-weight--bolder: bolder;
```

Comment:
This should probably not be necessary. 

Having a parent of `$font-weight--X` makes it easy to determine a child with `$font-weight--Y`.

While a keyword of `bolder` might let browsers' non-predictable behavior determine it and
some times browsers can make both texts appear the same weight.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2278/commits/f423c3772cb4e9bb2a88f09ccc44a553786ca63d